.github/workflows/run_quic_interop.yml

```diff
@@ -0,0 +1,37 @@
+name: "Run openssl quic interop testing"
+
+on:
+  workflow_run:
+    workflows: ["Build openssl interop container from master"]
+    types: [completed]
+  workflow_dispatch:
+
+jobs:
+  run_quic_interop:
+    strategy:
+      matrix:
+        tests: [http3, transfer, handshake, retry, chacha20]
+        servers: [quic-go, ngtcp2, mvfst, quiche, nginx, msquic, haproxy]
+      fail-fast: false
+    runs-on: ubuntu-latest 
+    steps:
+      - uses: actions/checkout@v4
+        with:
+         repository: 'quic-interop/quic-interop-runner'
+         fetch-depth: 0
+      - name: Install dependencies
+        run: |
+          pip install -r requirements.txt
+          sudo add-apt-repository ppa:wireshark-dev/stable
+          sudo apt-get update
+          sudo apt-get install -y tshark
+      - name: Patch implementations file
+        run: |
+          head -n -2 implementations.json > ./implementations.tmp
+          echo -e "},\n\"openssl\": {\n\"image\": \"quay.io/openssl-ci/openssl-quic-interop\",\n\"url\": \"https://github.com/openssl/openssl\",\n\"role\": \"client\"\n}\n}" >> ./implementations.tmp
```

Comment:
Can we ensure that `jq` is present in the runtime environment?  If so, it'd be a much cleaner tool for tweaking JSON documents than the above...

Reply:
yeah, I agree.  Jq is installed in the runner already, but the format of the interop code json file is a bit odd. Its a bare list of structs, rather than an array, so it I'm having a hard time figuring out the jq syntax for it directly.  let me take another stab at it

Reply:
Here's the command:

```sh
cat implementations.json | jq '. += {"openssl": {"image": "quay.io/openssl-ci/openssl-quic-interop", "url": "https://github.com/openssl/openssl", "role": "client"}}' > ./implementations.json
```